packages/bpk-component-rating/README.md

```diff
@@ -31,12 +31,15 @@ export default () => (
 | title     | string                | true     | -                 |
 | value     | number                | true     | -                 |
 | className | string                | false    | null              |
+| pill      | boolean               | false    | false             |
```

Comment:
Readme needs updating to use `RATING_TYPES` instead of `pill`.

Reply:
Yep just realised after I requested review 😆